app/code/Edu/CmsCreateSlider/Setup/Patch/Data/AddNewCmsProducts.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Edu\CmsCreateSlider\Setup\Patch\Data;
+
+use Magento\Catalog\Model\Product\Attribute\Source\Status;
+use Magento\Catalog\Model\ProductFactory;
+use Magento\Framework\App\Area;
+use Magento\Framework\App\State;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Class AddNewCmsPageTask2
+ * @package Edu\CmsCreateSlider\Setup\Patch\Data
+ */
+class AddNewCmsProducts implements
+    DataPatchInterface
+{
+    /**
+     * @var ProductFactory
+     */
+    protected $productFactory;
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    protected $moduleDataSetup;
+
+    /**
+     * @var State
+     */
+    private $state;
+
+    /**
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param ProductFactory $productFactory
+     * @param State $state
+     */
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup,
+        ProductFactory $productFactory,
+        State $state
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->productFactory = $productFactory;
+        $this->state = $state;
+    }
+
+    /**
+     * {@inheritdoc}
+     * @throws \Exception
+     */
+    public function apply()
+    {
+        $this->moduleDataSetup->startSetup();
+        $this->state->setAreaCode(Area::AREA_FRONTEND);
+        for ($i = 0; $i <= 15; $i++) {
+            $product = [
+                'sku' => 'SimpleProduct' . $i,
+                'name' => 'Simple Product ' . $i,
+                'attribute_set_id' => '4',
+                'website_ids' => [1],
+                'status' => Status::STATUS_ENABLED,
+                'visibility' => 4,
```

Comment:
It is best to avoid using hardcoded values as they might change, try to see if a constant for this value is available somewhere in core (and by constant for this value I don't mean a constant that happens to be equal to 4, but rather a constant that represents the visibility, or any other attribute you need)